x-pack/test/api_integration/apis/ml/jobs/force_start_datafeeds.ts

```diff
@@ -0,0 +1,248 @@
+  ): Promise<any> {
+    const { body } = await supertest
+      .post('/api/ml/jobs/force_start_datafeeds')
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(COMMON_REQUEST_HEADERS)
+      .send(requestBody)
+      .expect(expectedResponsecode);
+
+    return body;
+  }
+
+  const testDataList = [
+    {
+      testTitle: 'as ML Poweruser',
+      jobIds: [SINGLE_METRIC_JOB_CONFIG.job_id, MULTI_METRIC_JOB_CONFIG.job_id],
+      user: USER.ML_POWERUSER,
+      requestBody: {
+        datafeedIds: [
+          `datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`,
+          `datafeed-${MULTI_METRIC_JOB_CONFIG.job_id}`,
+        ],
+        start: 1454803200000, // Starts in real-time from Feb 7 2016 00:00
+      },
+      expected: {
+        responseCode: 200,
+        responseBody: {
+          [`datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`]: { started: true },
+          [`datafeed-${MULTI_METRIC_JOB_CONFIG.job_id}`]: { started: true },
+        },
+      },
+    },
+  ];
+
+  const invalidTestDataList = [
+    {
+      testTitle: 'as ML Poweruser with datafeed ID that does not exist',
+      jobIds: [SINGLE_METRIC_JOB_CONFIG.job_id],
+      user: USER.ML_POWERUSER,
+      requestBody: {
+        datafeedIds: [`invalid-datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`],
+        start: 1454803200000, // Feb 7 2016 00:00
+      },
+      expected: {
+        responseCode: 200,
+        responseBody: {
+          [`invalid-datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`]: { started: false },
+        },
+      },
+    },
+  ];
+
+  const testDataListUnauthorized = [
+    {
+      testTitle: 'as ML Unauthorized user',
+      user: USER.ML_UNAUTHORIZED,
+      jobIds: [SINGLE_METRIC_JOB_CONFIG.job_id, MULTI_METRIC_JOB_CONFIG.job_id],
+      requestBody: {
+        datafeedIds: [
+          `datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`,
+          `datafeed-${MULTI_METRIC_JOB_CONFIG.job_id}`,
+        ],
+        start: 1454803200000, // Feb 7 2016 00:00
+        end: 1455235200000, // Feb 12 2016 00:00
+      },
+      expected: {
+        responseCode: 403,
+        error: 'Forbidden',
+      },
+    },
+    {
+      testTitle: 'as ML Viewer',
+      user: USER.ML_VIEWER,
+      jobIds: [SINGLE_METRIC_JOB_CONFIG.job_id, MULTI_METRIC_JOB_CONFIG.job_id],
+      requestBody: {
+        datafeedIds: [
+          `datafeed-${SINGLE_METRIC_JOB_CONFIG.job_id}`,
+          `datafeed-${MULTI_METRIC_JOB_CONFIG.job_id}`,
+        ],
+        start: 1454803200000, // Feb 7 2016 00:00
+        end: 1455235200000, // Feb 12 2016 00:00
+      },
+      expected: {
+        responseCode: 403,
+        error: 'Forbidden',
+      },
+    },
+  ];
+
+  describe('force_start_datafeeds', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
```

Comment:
In this case we could run into issues with just deleting the `.ml-*` indices (see CI failures) and should explicitly delete the jobs:
```suggestion
    after(async () => {
        for (const job of testSetupJobConfigs) {
          await ml.api.deleteAnomalyDetectionJobES(job.job_id);
        }
        await ml.api.cleanMlIndices();
    });
```

Reply:
Done in https://github.com/elastic/kibana/pull/110961/commits/fa88781319a27a3ce24a458ed4745831f282b81e